client/my-sites/guided-transfer/guided-transfer.jsx

```diff
@@ -86,7 +86,11 @@ export default React.createClass( {
 				{ this.props.isEligibleForGuidedTransfer
 					? <div className="guided-transfer__content">
 						{ hostInfo
-							? <HostCredentialsPage hostSlug={ this.props.hostSlug } hostInfo={ hostInfo } />
+							? <HostCredentialsPage
+								siteId={ this.props.siteId }
+								siteSlug={ this.props.siteSlug }
```

Comment:
This goes to show why `connect` close to the component is nice, since we can skip passing the prop through three layers of the hierarchy.